nipoppy/workflow/bids_conv/run_dcm2bids.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import glob
+import os
+import shutil
+import subprocess
+from joblib import Parallel, delayed
+from pathlib import Path
+from typing import Iterable
+
+import numpy as np
+from bids.layout import parse_file_entities
+
+import nipoppy.workflow.catalog as catalog
+import nipoppy.workflow.logger as my_logger
+from nipoppy.workflow.utils import (
+    COL_CONV_STATUS,
+    COL_DICOM_ID,
+    DNAME_BACKUPS_DOUGHNUT,
+    FNAME_DOUGHNUT, 
+    load_doughnut,
+    save_backup,
+    session_id_to_bids_session,
+)
+
+#Author: nikhil153
+#Date: 07-Oct-2022
```

Comment:
I didn't know I wrote this!

Reply:
I think what Nikhil meant but that comment is that you can delete those lines :)